README.md

```diff
@@ -5,54 +5,9 @@
 
 *nRF Connect Boilerplate* provides a starting point for developing apps that can be launched by [nRF Connect for Desktop](https://github.com/NordicSemiconductor/pc-nrfconnect-core).
```

Comment:
Shall we change the link to pc-nrfconnect-launcher?

Reply:
Oh, I missed that earlier. Thanks for noticing. I created #33 for it.